src/H5AC.c

```diff
@@ -197,6 +197,8 @@ H5AC_term_package(void)
  *
  * Programmer:  John Mainzer, 1/10/17
  *
+ * Changes:     None.
```

Comment:
Please no!

Reply:
We've left the "Changes" sections alone in the cache code (they've been removed from almost everywhere else). We'll be updating the function header comments later this summer and we'll remove all the noise (authorship, changes, etc.)